x-pack/plugins/observability_solution/apm/server/routes/errors/get_error_groups/get_error_sample_details.ts

```diff
@@ -78,6 +77,7 @@ export async function getErrorSampleDetails({
   ] as const);
 
   const optionalFields = asMutableArray([
+    TRACE_ID,
```

Comment:
Revert this change and run the `group_id_samples.spec.ts` to reproduce the error.